src/core/issues-notes.js

```diff
@@ -141,22 +141,25 @@ export function run(conf, doc, cb) {
     issueBase = conf.issueBase;
   if ($ins.length) {
     if (conf.githubAPI) {
-      ghFetch(conf.githubAPI)
-        .then(function(json) {
-          issueBase = issueBase || json.html_url + "/issues/";
-          return fetchIndex(json.issues_url, {
-            // Get back HTML content instead of markdown
-            // See: https://developer.github.com/v3/media/
-            Accept: "application/vnd.github.v3.html+json",    
-          });
-        })
-        .then(function(issues) {
-          issues.forEach(function(issue) {
-            ghIssues[issue.number] = issue;
-          });
-          handleIssues($ins, ghIssues, issueBase);
-          cb();
-        }).catch(err => {
+      const asyncFetch = async () => {
+        const json = await ghFetch(conf.githubAPI);
+        const issues = await getIssues(json);
+        issues.forEach( (issue) => {
+          ghIssues[issue.number] = issue;
+        });
+        handleIssues($ins, ghIssues, issueBase);
```

Comment:
I wonder, if we are doing `handleIssues()` and `cb()` both here had in `.catch()`, can we make use of `.finally()`:

```Js
asyncFetch()
  .catch(err => pub("error", err.message))
  .finally(() => { 
     handleIssues($ins, ghIssues, issueBase)
     cb();
  })
```

Reply:
Yes you are right. ```handleIssues()``` and ```cb()``` are present in both ```then()``` and ```catch()```. We can avoid duplicacy here. Updating.